lib/full-site-editing/class-wp-rest-templates-controller.php

```diff
@@ -199,6 +203,11 @@ public function update_item( $request ) {
 			return new WP_Error( 'rest_template_not_found', __( 'No templates exist with that id.', 'gutenberg' ), array( 'status' => 404 ) );
 		}
 
+		if ( isset( $request['reverted_file_content'] ) && isset( $request['content'] ) && $request['reverted_file_content'] === $request['content'] ) {
```

Comment:
Should this go into separate `revert_item` handler?

Reply:
Probably 🤔 But in that case we need to call a custom route from the client side, don't we?